### ISSUE TITLE
Remove dependency on System.ValueTuple NuGet package when targeting .Net framework 4.7

### ISSUE THREAD

**Issue by Andrej-Peplinski** [dotnet/reactive#840]
This is to avoid having to add unnecessary dependencies in consuming projects.

**Comment by bartdesmet** [comment]
@onovotny, is this something that still needs attention?

This may bring a little pickle when we add overloads for e.g. `Zip` that use `ValueTuple<...>` to be symmetric with newer extension methods added to `IEnumerable<T>`. In particular:

```csharp
public static IObservable<(TFirst First, TSecond Second)> Zip<TFirst, TSecond>(this IObservable<TFirst> first, IObservable<TSecond> second);
```

**Comment by onovotny** [comment]
It's probably worth adding another build target for the first version of .NET that has ValueTuple in it.

**Comment by joshjje** [comment]
I third this.

**Comment by glopesdev** [comment]
Are there any plans to move forward on this? Is there a good reason to keep targeting net46 when `System.Reactive` requires `System.ValueTuple` anyway? In that case might as well just target net472.

**Comment by glopesdev** [comment]
I guess another way to look at it is to consider the `netstandard2.0` target as the compatibility layer for net47 applications (since `netstandard2.0` does include `System.ValueTuple`). In that case, maybe a possible "resolution" for the issue is to simply remove the `net46` target?

**Comment by bartdesmet** [comment]
@clairernovotny, let's address this before we hit 5.0 RTM. I'm happy to drop 4.6 and either add 4.7.2, or just converge on `.netstandard2.0` for downlevel support.

**Comment by clairernovotny** [comment]
I think we can set net472 as the new min esp since it's a major version bump. Having a net472 tfm is good for two reasons:

- Reduces binding redirect issues with netstandard libraries
- Has our WinForms and WPF support

**Comment by bartdesmet** [comment]
This has been addressed now and will be in Rx 5.0.
